examples/zephyr/lightdb_stream/pytest/test_sample.py

```diff
@@ -34,5 +34,5 @@ async def test_lightdb_stream(shell, device, wifi_ssid, wifi_psk):
             LOGGER.info("ts: {0}, temp: {1}".format(value['timestamp'], value['data']['sensor']['temp']))
             assert (value["data"]["sensor"]["temp"] == temp)
             temp += 0.5
-            if temp == 21.5:
+            if temp == 23.5:
```

Comment:
> Can we add tests for the new functionality? Should be as simple as extending the number of temperature readings we're checking (in examples/zephyr/lightdb_stream/pytest/test_sample.y) to ensure that we're covering the new functions.

@sam-golioth is this what you were thinking?

Reply:
Yep! We only added two new set operations, so `22.5` should suffice.

Reply:
Ahh, got it. I misread the test previously and thought it was checking each set operation twice.

Based on your comments, I believe that the intent is just to check that we get a single valid value for each set operation, right? If that's the case, couldn't the new value actually be `22.0`?

i.e.

1. JSON (sync): `20.0` <- initial value of `temp`
2. JSON (async): `20.5`
3. CBOR (sync): `21.0`
4. CBOR (async): `21.5`
5. (loop breaks at `22.0` but doesn't check this value)

Reply:
Ah yeah, that's right. We previously had a `lightdb_stream_set_int()` function that we were also testing, but it's been removed, probably forgot to update the test.

Reply:
I just changed it to `22.0`. I think this PR should be ready for review now.